src/dataServices/dynamoDbBundleService.ts

```diff
@@ -39,23 +39,42 @@ export class DynamoDbBundleService implements Bundle {
 
     private maxExecutionTimeMs: number;
 
+    readonly enableMultiTenancy: boolean;
+
     private static readonly dynamoDbMaxBatchSize = 25;
 
     // Allow Mocking DDB
-    constructor(dynamoDb: DynamoDB, supportUpdateCreate: boolean = false, maxExecutionTimeMs?: number) {
+    constructor(
+        dynamoDb: DynamoDB,
+        supportUpdateCreate: boolean = false,
+        maxExecutionTimeMs?: number,
+        { enableMultiTenancy = false }: { enableMultiTenancy?: boolean } = {},
```

Comment:
I am sadly confused by this syntax so we have 3 parameters and then an object to pass in and by default it is an empty object?

Why just not pass in a Boolean and default to false?

Reply:
Agreed cool good to know future plans :)